src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine one = new Excavator();
+        Machine two = new Bulldozer();
+        Machine three = new Truck();
```

Comment:
```suggestion
        Machine excavator = new Excavator();
        Machine bulldozer = new Bulldozer();
        Machine truck = new Truck();
```
take care about naming))

Reply:
Okay. Thank you !